test/integration/test_ibm_provider.py

```diff
@@ -73,37 +73,6 @@ def test_pass_unreachable_proxy(self, qe_token, qe_url):
             IBMProvider(qe_token, qe_url, proxies=proxies)
         self.assertIn("ProxyError", str(context_manager.exception))
 
-    def test_provider_init_non_auth_url(self):
```

Comment:
I've split off unit tests from this integration test file and added the removed test cases to a new file within `test/unit`